layers/meta-resin-jetson/recipes-containers/balena/balena_git.bbappend

```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```

Comment:
How is this broken? I remember this being fixed a while ago: resin-os/meta-resin@1d70610

Did you have any problems with this?

Reply:
I remember even after that commit the TX2 had the mentioned issue. I will give it a test.

Reply:
@floion this is still broken

Reply:
@telphan I remember we created an issue at upx about this? Would you reference it and maybe follow up with what they were requesting? I think it was a minimum test case.

Reply:
Uh wait. I found the issue: https://github.com/upx/upx/issues/130#issuecomment-338521057 . So we should mention this in the bbappend and make sure we remove it when the next version is released.

Reply:
@agherzan I cannot find the issue. Maybe we have not raised it after all.

Reply:
Included the issue in the bbappend.